numpy/core/src/umath/override.c

```diff
@@ -152,6 +155,9 @@ normalize_reduce_args(PyUFuncObject *ufunc, PyObject *args,
             }
             obj = PyTuple_GetSlice(args, 3, 4);
         }
+        if (i == 5 && obj == NoValue) {
+            continue;
```

Comment:
A comment saying this is the `initial` argument would be good.

Reply:
Sure. Any idea why the Windows build is failing?